docs/class_structure.md

```diff
@@ -0,0 +1,23 @@
+# Class structure in ODIES
+
+A major focus of ODIES is modularizing the components and system architecture so it's easier to construct and analyze complex hybrid power plants producing commodities for a variety of uses.
```

Comment:
I know line length isn't enforced in GreenHEART, but it would be nice if we avoided very long lines to be kinder to our future selves.

Reply:
I get what you're saying in general, especially for code. I'm a fan of single sentence lines for docs regardless of length, though I do generally have a pretty wide monitor. Are you thinking we should have a ~100 char length line limit for docs too?